WeakAurasOptions/CommonOptions.lua

```diff
@@ -1123,9 +1131,38 @@ local function PositionOptions(id, data, _, hideWidthHeight, disableSelfPoint, g
       order = 72,
       image = function() return "", 0, 0 end,
       hidden = function()
-        return IsParentDynamicGroup() or not (data.anchorFrameType == "SCREEN" or data.anchorFrameType == "UIPARENT" or data.anchorFrameType == "MOUSE" or IsGroupByFrame())
+        return
+          IsParentDynamicGroup() or
+          not (
+            data.anchorFrameType == "SCREEN" or
+            data.anchorFrameType == "UIPARENT" or
+            data.anchorFrameType == "MOUSE" or
+            data.anchorFrameType == "ACTION_BUTTON_SPELL" or
+            IsGroupByFrame()
+          )
       end,
     },
+    anchorFrameActionButtonSpell = {
+      type = "input",
+      width = WeakAuras.doubleWidth,
+      name = L["Spell"],
+      order = 72,
```

Comment:
(nit) this order attribute is equal to a corresponding one in an option above, so the order between them is indeterminate. obviously due to hidden attributes it ends up moot, but ideally the order is unique in any case